### PR TITLE
Vue beer component

### PR DESCRIPTION
**Pull request by matt-harris**
Adding Beer component and updating CSS styles.

### CODE REVIEW

File: src/components/Beer.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+
+<script>
+export default {
+  name: 'Beer',
+  props: {
+    beer: Object
+  },
+  computed: {
+    foodPairings: function() {
```
Comment (atrunelle) [inline src/components/Beer.vue:47]: This is exactly what compute property are intended for.

With ES6, you could remove the function completely:

```
 foodPairings() {
// body
}
```

File: src/components/Beer.vue [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      const items = this.beer.food_pairing;
+      return items.slice(0, 3);
+    }
+  }
+};
+</script>
+
+<style lang="scss">
+.beer {
```
Comment (atrunelle) [inline src/components/Beer.vue:56]: I am a big fan of BEM too 😍

**Review by atrunelle — APPROVED** [review]
Looks great! I like that you are using BEM, your code is clean and you are thinking of accessibility as well 👍

File: src/components/Beer.vue (hunk 1 above)
Comment (matt-harris) [reply src/components/Beer.vue:47]: Thanks Aurore, will get this added 👍

**Pull request merged**